core/src/avm1/globals/array.rs

```diff
@@ -674,70 +674,61 @@ fn qsort<'gc>(
         return Ok(());
     }
 
-    // Fast-path for 2 elements.
-    if let [(_, a), (_, b)] = &elements {
-        if compare_fn(activation, a, b, options)?.is_gt() {
-            elements.swap(0, 1);
+    // Stack for storing subarray boundaries (start and end).
```

Comment:
to be sure, it's an inclusive-inclusive range? Can you please add that to the comment?

Reply:
> it's an inclusive-inclusive range?

Yes, it is